api/backend/ecs/service_manager.go

```diff
@@ -57,6 +57,21 @@ func (this *ECSServiceManager) ListServices() ([]*models.Service, error) {
 	return services, nil
 }
 
+func (this *ECSServiceManager) ListServiceNames() ([]id.L0ServiceID, error) {
```

Comment:
You should update `ListServices` instead of adding this new function in this instance